### PR TITLE
#347 issue fixed

### LINKED ISSUE #347: Add a FlipBook
I am a GSSOC'22 participant and want to add a flipbook project which is made using HTML, CSS and JS 

### Screenshots

image
image

@VidhiBhatt01 Kindly assign me this issue

### PR DESCRIPTION
**Pull request by shivam123-dev**
# Description

Added a responsive flipbook project
Fixes #347 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK
image
image
image
image

### CODE REVIEW

**Review by VidhiBhatt01 — APPROVED** [review]

File: Flipbook/turn.js [hunk 1]
```diff
@@ -0,0 +1,1933 @@
...
+ * modification, are permitted provided that the following conditions
+ * are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright
+ * notice, this list of conditions and the following disclaimer.
+ *
+ * 2. Any redistribution, use, or modification is done solely for personal 
+ * benefit and not for any commercial purpose or for monetary gain.
+ * 
+ **/
```
Comment (arpit456jain) [inline Flipbook/turn.js:18]: @shivam123-dev i don't think u made this from ur own

**Comment by arpit456jain** [comment]
https://codepen.io/adi9090/pen/XWedXEJ

File: Flipbook/turn.js (hunk 1 above)
Comment (shivam123-dev) [reply Flipbook/turn.js:18]: No @arpit456jain sir I didn't made the turn.js API. With the help of this API, I just added the turning animation. This API (turn.js) has been downloaded from its [original source](http://turnjs.com/).

Comment (shivam123-dev) [reply Flipbook/turn.js:18]: So, I request you to kindly remove the label of plagiarism.

**Pull request merged**